data/zilliqa/assets/zil133qfql0r3k33j8te8kpaxagdemupfd9wvh88ka/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ZAME Coin",
+  "symbol": "ZAME",
+  "web": "https://thezame.app",
+  "supply": "1725000",
+  "email": "info@thezame.app",
+  "whitepaper": "https://thezame.app/wp-content/uploads/2021/12/ZAME-Coin-Whitepaper.pdf",
+  "publicTeam": true,
+  "links": {
+    "telegram": "https://t.me/zametoken",
+    "twitter": "https://twitter.com/thezameapp",
+    "linkedin": "https://www.linkedin.com/in/saayanchoudhury/",
```

Comment:
no pic

Reply:
Sorry, didn't understand. What needs to be changed - could you please mention, thanks. The logo.png is there

Reply:
this linkedin isn't good enough to deserve the public team prop